app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
```

Comment:
- [ ] It would be better if the application could run without any errors, however after creating and migrating the database, I tried to run the `rails s` but got `ArgumentError (wrong number of argument(given 3, expected [1..2]). So i commented out line 3 `@users = User.all` before the server run successfully.

Reply:
Thank you @addod19, yes you're right I did the same, and works great, now all required changes are done 😄